applications/notifications/services.py

```diff
@@ -0,0 +1,43 @@
+from django.db import transaction
+import environ
+from onesignal_sdk.client import Client as OneSignalClient
+
+from .models import Notification
+
+env = environ.Env()
+
+
+class NotificationSender(object):
+    client = OneSignalClient(app_id=env('ONESIGNAL_APP_ID'), rest_api_key=env('ONESIGNAL_API_KEY'))
+
+    def send_notification(self, emails, data, message):
+        body = self._body(emails, data, message)
+        self.client.send_notification(body)
+
+    def _body(self, emails, data, message):
+        return {
+            'filters': self._create_filter_by_emails(emails),
+            'data': data,
+            'contents': {'en': message},
+        }
+
+    def _create_filter_by_emails(self, emails):
+        email_list = emails if type(emails) is list else [emails]
+
+        def email_filter(email):
+            return {'field': 'tag', 'key': 'email', 'relation': '=', 'value': email}
```

Comment:
if you're just adding the email why not define the dict as a variable and do `my_dict['value'] = email`?

Reply:
I update the PR description adding an example about what I'm trying to do here.